chandra_aca/star_probs.py

```diff
@@ -213,50 +186,67 @@ def prob_n_acq(star_probs):
     return n_acq_probs, np.cumsum(n_acq_probs)
 
 
-def acq_success_prob(date=None, t_ccd=-19.0, mag=10.0, color=0.6, spoiler=False, halfwidth=120):
+def acq_success_prob(date=None, t_ccd=-19.0, mag=10.0, color=0.6, spoiler=False, halfwidth=120,
+                     model='default-for-date'):
```

Comment:
Not sure if I need a way to control the model from `t_ccd_warm_limit`.  It is probably OK to just have that use the default.  A problem now in the aca_lts_eval code is that it is using the cached value of the calculated temperature from the time of the first calculation.  So if I ran this code now over the full interval (which starts last August), for any star combinations that were seen before 01-Apr-2018, I'd be getting the old predictions, even if the combinations were seen in the future.  I had a plan to update that cache code anyway, or I could just run the aca_lts_eval code starting at 01-Apr-2018 instead of back to August.

Reply:
That's already done.